### ISSUE TITLE
[Bug]: Ant-size program font/menu display scaling on a high-resolution display. Compatibility scaling settings don't do anything

### ISSUE THREAD

**Issue by AnthonyBronislavaGonzales** [Sigil-Ebook/Sigil#734]
### Bug Description

Menus and menu font are tiny and headache-inducing on a display larger than 1920x1080. Typically this problem can be solved by compatibility settings, i.e. high DPI settings.

System scaling by Application: no change, still tiny menus and fonts
System scaling by System: no change, except causes flickering (the latter is not atypical, no one uses this setting)
System scaling by System (enhanced): typically this setting fixes scaling issues, but with Sigil you get a completely bugged screen

### Platform (OS)

Windows (Default)

### OS Version / Specifics

Win 10

### What version of Sigil are you using?

current

**Comment by dougmassay** [comment]
Sigil has worked fine on high resolution monitors for many versions for many users. I myself use it with 3840x2160 monitors on both Windows and Linux with differing scale factors with no perceived problems. There have also been zero reports of "ant-sized" menu fonts on our user forums.

**Comment by dougmassay** [comment]
Make sure you're not using any system-wide QT_* environment variables meant to affect Qt6's automatic high dpi handling. Specifically QT_ENABLE_HIGHDPI_SCALING=0 will completely disable any high dpi scaling.

The following list of environment variables will affect the Qt6 bundled with Sigil's ability to automatically manage high dpi settings.

QT_FONT_DPI (legacy variable for backward compatability)
QT_SCALE_FACTOR
QT_SCALE_FACTOR_ROUNDING_POLICY
QT_SCREEN_SCALE_FACTORS
QT_USE_PHYSICAL_DPI

Have you changed the default system font and/or font size in Sigil's preferences?

In short, it's going to be difficult for me to troubleshoot an issue I (and other users) can't replicate.

**Comment by AnthonyBronislavaGonzales** [comment]
Thanks for the suggestions Doug. I understand the difficulty you state. I will look into these items and see if I can find a solution, or if not, if I can find more information for you. No one else has encountered this scaling issue?

**Comment by dougmassay** [comment]
If they have encountered them, they've not reported them. We had some issues early on with high dpi/high resolution on Windows when Qt5 first provided settings for them (mostly multi-monitor stuff)., but haven't heard anything for a long time. The transition to Qt6's automatically handling of the scaling has been fairly seamless so far.

**Comment by dougmassay** [comment]
For the record: if I set the QT_SCALE_FACTOR environment variable to some thing less than 1 (like .75), I get pretty small Sigil fonts. And none of the high dpi settings overrides that you mentioned in your first post would affect it.

**Comment by kevinhendricks** [comment]
Any updates as to what is causing your issues on your machine?

**Comment by kevinhendricks** [comment]
I am closing this as there is no Sigil bug that we can reproduce. If you find out what is causing the issue to happen or want help tracking it down.  Post here and we would be happy to reopen this issue.

**Comment by AnthonyBronislavaGonzales** [comment]
FWIW I finally figured this out. Preferences ---> Appearance ---> Main UI --- > Main UI Font ---> something bigger ---> restart

That solves it.
